tutorial/lesson-1/README.md

```diff
@@ -43,15 +44,27 @@ to test the View or UI.
 
 ## File and Directory Organization
 
-The files for this lesson are in the `lesson-1/app` directory. The directory
-is laid out as indicated in sub-directories.
+WABuR projects follow a suggested layout. Not all directories will be present
+in this lesson as some wabur and other runners will supply if not found in the
+directories.
+
+The files for this lesson will be in the `lesson-1/blog` directory which will
+be laid out as indicated.
 
 ```
-app
+blog
+├── config
+|   └── wabur.conf
+|   └── opo.conf
+|   └── opo-rub.conf
 ├── lib
-|   └── ui
-|       └── entry.rb
-└── pages
+|   └── ui_controller.rb
+└── site
+    └── assets
+        └── css
+        └── fonts
+        └── index.html
```

Comment:
an `index.html` within the `assets` directory?

Reply:
oops